patterns/services-subscriber-only-section.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Title: Services - Subscriber only section
+ * Slug: twentytwentyfive/services-subscriber-only-section
+ * Categories: call-to-action, featured, services
+ * Description: A subscriber-only section highlighting exclusive services and offerings.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"var:preset|spacing|50","padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70)"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|70","left":"var:preset|spacing|70"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"verticalAlignment":"center"} -->
+<div class="wp-block-column is-vertically-aligned-center"><!-- wp:heading {"fontSize":"xx-large"} -->
+<h2 class="wp-block-heading has-xx-large-font-size">Subscribe to get unlimited access</h2>
+<!-- /wp:heading -->
+
+<!-- wp:list {"className":"is-style-checkmark-list","style":{"spacing":{"margin":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"}}}} -->
+<ul style="margin-top:var(--wp--preset--spacing--40);margin-bottom:var(--wp--preset--spacing--40)" class="wp-block-list is-style-checkmark-list"><!-- wp:list-item {"fontSize":"medium"} -->
```

Comment:
It looks like we can set the padding-left of this list to x-small to make it match the design.

Reply:
@juanfra Thanks for reviewing. I have set the padding-left of this list to x-small. Please take a look and let me know your thoughts.